10_3/ex01bonus.js

```diff
@@ -0,0 +1,34 @@
+function resultDivs(res) {
+  const result = [2, 3, 5, 10].map(e => res / e);
+  return result;
+}
+
+function sumDivs(sum) {
+  console.log(sum.reduce((acc, cur) => acc + cur));
+}
+
+function onError(error) {
+  console.log(error);
+}
+
+const mathPromise = () => {
+  const arrayNums = [];
+  for (let i = 0; i < 10; i += 1) {
+    arrayNums.push(Math.round(Math.random() * 50));
+  }
+  const result = arrayNums.reduce((acc, cur) => acc + cur ** 2);
```

Comment:
Pedrão, do jeito que esta a soma não eleva ao quadrado o elemento 'arrayNums[0]'. Mas isso é facilmente ajustado inserindo um initial value para o seu reduce ;)
```suggestion
  const result = arrayNums.reduce((acc, cur) => acc + cur ** 2), 0;
```